### DISCUSSION TITLE
Weird artifacting with the launcher

### DISCUSSION THREAD

**Discussion by Erocktastic** [runelite/runelite#16506] [Q&A] [answered]
Has this happened to anyone? I just got a new graphics card and everything else is running smoothly except for runelite. It is having a [weird pixel artifacting](https://i.gyazo.com/2d40c2950abba0a048ec4a1b6fa25360.png) that just gets worse the more I interact with the program. Running runelite through the graphics card without any drivers installed didn't cause this issue, but once I installed AMD radeon with the up-to-date drivers for the graphics card then the artifacting would happen. Tried messing around with settings in both runelite and graphics cards settings but neither led to results.

Graphics Card: AMD Radeon RX 6700 XT
Driver version: 22.40.43.07-230326a-390319C-AMD-Software-Adrenalin-Edition
AMD Driver version: 31.0.14043.7000

**Comment by geheur** [comment, marked as answer]
Try turning off amd image sharpening.

**Reply by Erocktastic** [reply]
My Runelite specific settings for image sharpening were overriding my global settings of keeping it turned off and this was the issue. Thank you!
